test/functional/users_controller_test.rb

```diff
@@ -322,6 +322,7 @@ class UsersControllerTest < ActionController::TestCase
   end
 
   test "#login renders login page" do
+    session.clear
```

Comment:
Instead of undoing the setup method, it might be better to try and move tests relying on this into a context that redefines `setup_user` to an empty method.